### DISCUSSION TITLE
basic_content/sizeof/

### DISCUSSION THREAD

**Discussion by giscus** [Light-City/light-comments#7] [Announcements]
# basic_content/sizeof/

Stories About C Plus Plus

https://light-city.github.io/basic_content/sizeof/

**Comment by YLZei** [comment]
文章全是问题，把人恶心坏了，你可别出来祸害人了

**Reply by Light-City** [reply]
虚心接受你的建议，请提出哪里有问题？ 说话别这么冲，哪里惹你了吗？
